mmeds/config.py

```diff
@@ -19,7 +19,11 @@
     ROOT = Path(mmeds.__file__).parent.resolve()
     HTML_DIR = Path(html.__file__).parent.resolve()
     STORAGE_DIR = Path(resources.__file__).parent.resolve()
-    DATABASE_DIR = Path().home() / 'mmeds_server_data'
+    # For some reason my laptop can't find Path().home()
+    if gethostname() == 'fedora-y70':
```

Comment:
I'd suggest adding logging in case someone else runs into this problem. Something like: `if not Path().home().exists():` log "Home directory not found, try hard coding host name here"

Reply:
If the path doesn't exist they'll get an error that takes them right to this line, so I'm not sure the logging would really add much. That's also not exactly how the error plays out. It tries to look for apache's home directory rather than mine on my laptop for some reason

Reply:
Ah I see, well I like the extra comments you added so I'd agree, logging not necessary here.